pages/introduction.vue

```diff
@@ -54,9 +51,65 @@
         transition.value = "slide-next";
         step.value++;
     };
+
+    setTimeout(function(){if(step.value == 0){nextStep();}},8000);
+
 </script>
 
 <style>
+
+.move-in1{
+    animation: movein 2s linear;
+}
+
+.move-in2{
+    animation: moveindelayed 4s linear;
```

Comment:
```suggestion
    animation: movein 4s linear;
```

Reply:
das gleiche wie bei den Abstufungen. Da ich mit animation-delay und "start"-opacity 0 Probleme nach den Animationen hatte, habe ich separate Animation erstellt, die durch die Abstufungen "getimed" werden

Reply:
Hä, ich das sollte mit dem timing eigentlich nichts zu tun haben wie es grade ist....
Außerdem ist statt "liner" "ease-in-out" schöner

Reply:
wenn die animation 4s geht, der prozess aber erst bei 75% beginnt, dann ist davor 3s delay